bin/hdfcoinc/pycbc_page_foreground

```diff
@@ -99,11 +99,15 @@ if args.output_file.endswith('.html'):
     mass2 = fortrigs.get_bankfile_array('mass2')
     spin1z = fortrigs.get_bankfile_array('spin1z')
     spin2z = fortrigs.get_bankfile_array('spin2z')
-    time1 = fortrigs.get_coincfile_array('time1')
-    time2 = fortrigs.get_coincfile_array('time2')
+    time = fortrigs.get_end_time()
 
     if h_num_rm is not None and h_iterations is not None \
             and h_num_rm >= 0 and h_iterations != 0:
+        # FIXME: Currently heirarchical removal is not in the multi-detector
+        #        code. If this functionality is added, and we want to make this
+        #        table for heirarchically removed triggers, we would need to
```

Comment:
heirarchically --> hierarchically (and the same a couple of lines above)

Reply:
Fixed